### PR TITLE
Feature/prepare deployment

### PR DESCRIPTION
**Pull request by MichaelSchubertTNG**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+### Initial setup of database
+As a superuser on the database instance, run
+```
+create user betten;
+alter role "betten" with login;
+alter user betten with password '<password>';
+create database betten;
+\connect betten
+create extension "uuid-ossp"
+```
```
Comment (c7h) [inline backend/README.md:15]: with the flyway integration, this seems to be obsolete.

File: backend/local_deployment/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+version: '3.2'
+
+services:
+  db:
+    build: ./postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: postgres
```
Comment (c7h) [inline backend/local_deployment/docker-compose.yml:8]: we should put this in an environment file and create a environment.template file. This way, chances are lower to forget changing this for production.

File: backend/local_deployment/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      - 8080:8080
+  flyway:
+    restart: on-failure
+    image: trackyourbed-flyway
+    network_mode: host
+    depends_on:
+      - db
+    environment:
+      FLYWAY_USER: betten
+      FLYWAY_PASSWORD: betten
```
Comment (c7h) [inline backend/local_deployment/docker-compose.yml:25]: see comment above.

File: backend/local_deployment/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    environment:
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - 5432:5432
+  service:
+    image: trackyourbed
+    restart: always
+    network_mode: host
+    environment:
+      DB_HOST: 127.0.0.1
```
Comment (c7h) [inline backend/local_deployment/docker-compose.yml]: Would the container name `db` work here? In docker, it should resolve to the internal docker ip for the database server.

**Review by c7h — CHANGES_REQUESTED** [review]
Minor changes in the docker compose file and some questions about README.

File: backend/local_deployment/docker-compose.yml (hunk 4 above)
Comment (MichaelSchubertTNG) [reply backend/local_deployment/docker-compose.yml]: Yes that works, so then we can get rid of the network_mode as well, thank you!

File: backend/README.md (hunk 1 above)
Comment (MichaelSchubertTNG) [reply backend/README.md:15]: No, flyway acts on the initialized betten database

File: backend/local_deployment/docker-compose.yml (hunk 2 above)
Comment (MichaelSchubertTNG) [reply backend/local_deployment/docker-compose.yml:8]: This docker-compose is explicitly only for the local setup. The productive deployment will be with kubernetes

**Comment by atomdampflok** [comment]
don't merge right now, I messed something up with the commits

**Review by c7h — APPROVED** [review]

**Pull request merged**